django/core/files/storage.py

```diff
@@ -416,7 +424,7 @@ def get_modified_time(self, name):
 
 
 def get_storage_class(import_path=None):
-    return import_string(import_path or settings.DEFAULT_FILE_STORAGE)
+    return import_string(import_path or django_settings.DEFAULT_FILE_STORAGE)
```

Comment:
Changing `settings` to `django_settings` is really disruptive and unrelated, please revert it.

Reply:
Reverted, changed StorageHandler.settings to StorageHandler.backends